cmd/postgres_exporter/postgres_exporter.go

```diff
@@ -4,56 +4,44 @@ import (
 	"crypto/sha256"
 	"database/sql"
 	"errors"
-	"flag"
 	"fmt"
 	"io/ioutil"
 	"math"
 	"net/url"
 	"os"
 	"regexp"
+	"runtime"
 	"strconv"
+	"strings"
 	"sync"
 	"time"
 
 	"github.com/blang/semver"
-	_ "github.com/lib/pq"
+	"github.com/lib/pq"
 	"gopkg.in/yaml.v2"
 
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"github.com/prometheus/common/log"
 	"github.com/prometheus/common/version"
-
-	"strings"
+	"gopkg.in/alecthomas/kingpin.v2"
 
 	"github.com/percona/exporter_shared"
 )
 
+// Version is set during build to the git describe version
+// (semantic version)-(commitish) form.
+var Version = "0.0.1"
+
 var (
-	showVersion = flag.Bool(
-		"version", false,
-		"Print version information.",
-	)
-	listenAddress = flag.String(
-		"web.listen-address", getStringEnv("PG_EXPORTER_WEB_LISTEN_ADDRESS", ":9184"),
-		"Address to listen on for web interface and telemetry.",
-	)
-	metricsPath = flag.String(
-		"web.telemetry-path", getStringEnv("PG_EXPORTER_WEB_TELEMETRY_PATH", "/metrics"),
-		"Path under which to expose metrics.",
-	)
-	disableDefaultMetrics = flag.Bool(
-		"disable-default-metrics", getBoolEnv("PG_EXPORTER_DISABLE_DEFAULT_METRICS", false),
-		"Do not include default metrics.",
-	)
-	queriesPath = flag.String(
-		"extend.query-path", getStringEnv("PG_EXPORTER_EXTEND_QUERY_PATH", ""),
-		"Path to custom queries to run.",
-	)
-	onlyDumpMaps = flag.Bool(
-		"dumpmaps", false,
-		"Do not run, simply dump the maps.",
-	)
+	listenAddress          = kingpin.Flag("web.listen-address", "Address to listen on for web interface and telemetry.").Default(":9187").Envar("PG_EXPORTER_WEB_LISTEN_ADDRESS").String()
```

Comment:
That would require changes in pmm-managed's `postgresExporterConfig`.

Reply:
Oh. That's why it doesn't work... I'll fix that.